.github/workflows/release.yaml

```diff
@@ -50,3 +53,13 @@ jobs:
           args: release --rm-dist --timeout 1h
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update latest tag
+        uses: silverlyra/latest-image-action@v0.1.0
```

Comment:
I'm a bit awry in using this as I have long-term maintenance concerns, but I think it's OK for now until we think about a more permanent solution.